lib/associations/belongs-to-many.js

```diff
@@ -445,6 +445,12 @@ BelongsToMany.prototype.injectGetter = function(obj) {
       throughWhere = {};
       throughWhere[association.foreignKey] = instance.get(association.source.primaryKeyAttribute);
 
+      if (options.through && options.through.where)
```

Comment:
We should probably move this after  `.scope` so through can overwrite the scope if needed